README.md

```diff
@@ -124,6 +124,39 @@ const mySteps: TourStep[] = [{
 
 You can also find a complete example [here](example/).
 
+## Built-in Helper Components
+
+You can take advantage of the already built-in customizable components. For example, our [TourBox] (https://stackbuilders.github.io/react-native-spotlight-tour/docs/build/#tourbox) component can be used as a tooltip container for each step.
+
+
+```tsx
+const tourSteps: TourStep[] = [{
+    alignTo: Align.SCREEN,
+    position: Position.BOTTOM,
+    render: props => (
+      <TourBox
+        title="Tour: Customization"
+        titleStyle={{ 
+          fontFamily: 'Roboto', 
+          color: '#90EE90', 
+          fontWeight: 'bold'
+        }}
+        backText="Previous"
+        nextText="Next"
+        {...props}
+      >
+        <Text>
+          {"This is the third step of tour example.\n"}
+          {"If you want to go to the next step, please press "}<BoldText>{"Next.\n"}</BoldText>
+          {"If you want to go to the previous step, press "}<BoldText>{"Previous.\n"}</BoldText>
+        </Text>
+      </TourBox>
+    ),
+  }]
+```
+
+The [SpotlightTourProvider](https://stackbuilders.github.io/react-native-spotlight-tour/docs/build/#spotlighttourprovider) also allows you to customize the tour overlay through its [overlayColor](https://stackbuilders.github.io/react-native-spotlight-tour/docs/build/interfaces/SpotlightTourProviderProps.html#overlaycolor) and [overlayOpacity](https://stackbuilders.github.io/react-native-spotlight-tour/docs/build/interfaces/SpotlightTourProviderProps.html#overlayopacity) props.
```

Comment:
Maybe we can add a subsection for this. Something like:
```md
### Tour customization

The [SpotlightTourProvider](...) also allows you to customize...
```

You can also mention that these customizations can be configured on the `< SpotlightTourProvider>` for all steps, and they can also be overridden on the configuration of each step if needed 🙂